functions-and-scope.js

```diff
@@ -72,6 +103,17 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+function highestGrade(grades) {
+    let highGrade = 0
```

Comment:
Het is geen goed voorbeeld, maar ik wil het je wel meegeven:
Wanneer je een array meekrijgt ([-1, -3, -2]) met alleen maar negatieve getallen, dan zal de functie altijd 0 teruggeven. Het is daarom verstandig om altijd de eerste waarde in de array te pakken:
```suggestion
    let highGrade = grades[0];
```

Reply:
Dank je wel voor het nakijken. Ik heb je opmerkingen overgenomen en aangepast.